src/detect-hostbits.c

```diff
@@ -1240,11 +1082,9 @@ void HostBitsRegisterTests(void)
     UtRegisterTest("HostBitsTestSig03", HostBitsTestSig03, 0);
 #endif
     UtRegisterTest("HostBitsTestSig04", HostBitsTestSig04);
-    UtRegisterTest("HostBitsTestSig05", HostBitsTestSig05);
 #if 0
     UtRegisterTest("HostBitsTestSig06", HostBitsTestSig06, 1);
 #endif
```

Comment:
@victorjulien as a part of clean up, should we also removed tests inside `#if 0... #endif` blocks or could they be useful later on?

Reply:
If there is no comment explaining why its commented out it can be removed. If there is a comment it will depend on what it says.

Reply:
@TheKharleeci could you please check the commented tests too as per Victor's suggestions?

Reply:
There is no comment explaining why it was commented out

Reply:
HostBitsTestSig03 is also commented out. Should I remove it too

Reply:
Yes, please. Any tests in any of the files that you're cleaning up that have been commented without an explicit reason, remove them.